app/models/vm_reconfigure_task.rb

```diff
@@ -39,12 +39,16 @@ def self.build_message(options, key, message, modifier = nil)
 
   def self.build_disk_message(options)
     if options[:disk_add].present?
-      disk_sizes = options[:disk_add].collect { |d| d["disk_size_in_mb"].to_i.megabytes.to_s(:human_size) + ", Type: " + d["type"].to_s }
+      disk_sizes = options[:disk_add].collect { |d| get_disk_size(d) }
       "Add Disks: #{options[:disk_add].length} : #{disk_sizes.join(", ")} "
     end
   end
   private_class_method :build_disk_message
 
+  def self.get_disk_size(disk)
```

Comment:
looking good

would be nice to avoid `get_`, possibly:

```suggestion
  def self.disk_size_message(disk)
```

Reply:
Made changes as suggested.  thanks